classes/log/PKPSubmissionEventLogEntry.inc.php

```diff
@@ -58,9 +58,6 @@
 define('SUBMISSION_LOG_REVIEW_READY',		0x40000018);
 define('SUBMISSION_LOG_REVIEW_CONFIRMED',		0x40000019);
 
-// Deletion of the last revision of a file
-define('SUBMISSION_LOG_LAST_REVISION_DELETED', 	0x50000003);
```

Comment:
If this log entry type might still exist in the database, this should probably be preserved (along with the related locale key) for historical purposes. Drop a note here that it's deprecated.